### PR TITLE
Added Longest Common Subsequence in Java & C++

### LINKED ISSUE #1262: Add Longest Common Subsequence in C++
See requirements: https://sample-programs.therenegadecoder.com/projects/longest-common-subsequence/
Add snipper here: https://github.com/TheRenegadeCoder/sample-programs/tree/master/archive/c/c-plus-plus

### PR DESCRIPTION
**Pull request by Shubham230198**
Congrats on taking the first step to contributing to the Sample Programs repository maintained by [The Renegade Coder][1]! For simplicity, please make sure that your pull request includes one and only one contribution.

## Complete the Applicable Sections Below

Find which section best describes your pull request and make sure you fill it out. To start, let us know which issue you've fixed.

- [x] I fixed #1262 

### Code Snippets

- [ ] I named the pull request using `Added/Updated <Sample Program> in <Language>` format
- [x] I created/updated the language README
  - [x] I added the sample program name to the README
  - [x] I added fun facts (i.e. debut, developer, typing, etc.)
  - [x] I added reference link(s) to the README
  - [x] I added solution citations when necessary (see [plagiarism][2])

### Documentation

- [ ] I named the pull request using `Added/Updated <Sample Program> in <Language> Article` format
- [ ] I followed the applicable article template
  - [ ] I added/updated a [language article][4] (i.e. The Python Programming Language)
  - [ ] I added/updated a [code snippet article][3] (i.e. Hello World in Perl)
  - [ ] I added/updated a [project article][5] (i.e. Fizz Buzz in Every Language)

### Testing

- [ ] I named the pull request using `Added/Updated <Language>/<Project> Testing` format
- [ ] I followed the testinfo template, if applicable

## Notes

Feel free to describe what you added or updated.

[1]: https://therenegadecoder.com/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#plagiarism
[3]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/CODE_ARTICLE_TEMPLATE.md
[4]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/LANGUAGE_ARTICLE_TEMPLATE.md
[5]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/PROJECT_ARTICLE_TEMPLATE.md

### CODE REVIEW

File: archive/c/c-plus-plus/longest-common-subsequence.cpp [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#include<iostream>
+#include<algorithm>
+#include<vector>
+#include<string>
+using namespace std;
+
+//The function to convert the input strings into the required array, by using delimiter.
+vector<string> splitStrings(string str) 
+{   
+	string word = "";
```
Comment (jrg94) [inline archive/c/c-plus-plus/longest-common-subsequence.cpp]: Indentation is a bit inconsistent here. Any chance you could run some formatting on these files?

Comment (Shubham230198) [reply archive/c/c-plus-plus/longest-common-subsequence.cpp]: ok I will do that.

**Comment by Shubham230198** [comment]
Done the changes, you requested.

File: archive/c/c-plus-plus/lcs.cpp [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+
+//Driver code.
+int main(int argc,char *argv[]) 
+{   
+    	//In case there is no input, empty input or only single input.
+    	if (!(argc> 2 && std::string(argv[1]) != "" && std:: string(argv[2])!=""))                  
+    	{
+        	cout << "Usage: please provide two lists in the format “1, 2, 3, 4, 5”" << endl;
```
Comment (jrg94) [inline archive/c/c-plus-plus/lcs.cpp]:
```suggestion
        	cout << "Usage: please provide two lists in the format \"1, 2, 3, 4, 5\"" << endl;
```

**Pull request merged**